wiki/Help_Centre/Performance/en.md

```diff
@@ -0,0 +1,110 @@
+
+If you're using one, you might want to adjust the DPI setting of your mouse, too. Higher DPI settings provide more accurate positioning to the game client and may smooth out any jitters.
+
+We also suggest ensuring your peripherals are clean and free from dirt or debris. A string of hair or bit of dust in front of a mouse's optical sensor can do very strange things to the way a cursor moves.
+
+### What is raw input, and how do I turn it on?
+
+**Raw input forces the game to use a cleaner and more direct means to access the information from your peripherals. This generally increases the accuracy of the output from the device.**
+
+That feeling of increased "accuracy" from turning on raw input is usually because your mouse will no longer "accelerate." Mouse acceleration is basically a way of making the cursor move faster to travel a farther distance (i.e., accelerate) when you speed-up your mouse movements. Most people like this feature when using a computer normally, but when playing osu!, it can cause unpredictable, and jumpy movement. 
+
+*Notice: If you're using a tablet, turning on raw input may cause the cursor to be stuck in the corner of the screen. This is due to the way tablets are designed, as they use absolute positioning (unlike computer mice, which use relative positioning)*
+
+To enable raw input, do the following:
+
+1. Open osu!.
+2. Enter the `Options` menu or press `Ctrl` + `O`.
+3. Type `raw` into the quick search to jump straight to the option.
+4. Click the `Raw input` radio button.
+
+## I'm playing from a tablet and my cursor feels too slow!
+
+**You can solve this problem by defining a smaller tablet area for your device.**
+
+On most Wacom tablets, this is done via the `Mapping` tab of the `Wacom Tablet Properties` application.
+
+![Wacom Tablet Properties screen showing Mapping options](img/Tabarea.png "Interface for changing tablet area on Wacom devices")
+
+You'll need to consult your tablet's manufacturer to find out how to set your tablet area for your particular peripheral.
+
+All osu! branded tablets support raw input by default, and may have their sensitivity easily adjusted on the fly by using using the mouse sensitivity slider. Unfortunately,<!--as of ____,--> osu! branded tablets are no longer being officially sold or manufactured.
+
+### My songs list keeps scrolling!
+
+**This is usually due to a faulty input device connected to your computer. Try unplugging any controllers or joysticks.**
+
+Applications which emulate or change keybindings such as *Xpadder* or *X-Mouse Button Control* may also cause these issues. Try disabling them if you use them for other games.
+
+This can also happen due to issues with the numpad keys, as they can be used to scroll the song selection list. Pressing the `NumLock` key to disable the numpad and then pressing either `9` or `3` on the numpad will fix the issue. You should also check for any jammed or damaged buttons on your peripherals while doing this.
```

Comment:
8 and 2 on the numpad can also fix this issue, not just 9 and 3 (can be determined based on how fast the list is scrolling). Pressing the up/down arrow keys themselves can also fix this in the case where 8 and 2 would have worked. Might be a good idea to include either one of the two? Not entirely sure how to word it though.

Reply:
I think a note including that information would would work fine here